pkg/provision/buildroot.go

```diff
@@ -151,6 +151,11 @@ WantedBy=multi-user.target
 		return nil, err
 	}
 
+	// because currently we install docker differently in kic, it is harmless to do this step for VM too so the code by the same
```

Comment:
I wouldn't go so far as claiming harmless - it does add startup latency and room for failure in a place where it isn't necessary.  I'm not convinced we should call this in the VM, but could easily be swayed by a good comment.

// We have possibly changed the Docker file, inform systemctl of this fact.